packages/text-area/src/TextArea.tsx

```diff
@@ -0,0 +1,278 @@
+import React, { useState, useMemo } from 'react';
+import { IdAllocator, Either, createDataProp } from '@leafygreen-ui/lib';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { spacing, fontFamilies } from '@leafygreen-ui/tokens';
+
+const idAllocator = IdAllocator.create('textarea');
+const textAreaProp = createDataProp('area-selector');
+
+export const State = {
+  None: 'none',
+  Error: 'error',
+} as const;
+
+export type State = typeof State[keyof typeof State];
+
+const Mode = {
+  Light: 'light',
+  Dark: 'dark',
+} as const;
+
+type Mode = typeof Mode[keyof typeof Mode];
+
+const containerStyles = css`
+  display: flex;
+  flex-direction: column;
+`;
+
+const labelStyle = css`
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 16px;
+  padding-bottom: 4px;
+`;
+
+const descriptionStyle = css`
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: normal;
+  padding-bottom: 4px;
+  margin-top: 0px;
+  margin-bottom: 0px;
+`;
+
+const textAreaContainer = css`
+  margin-top: 4px;
+  position: relative;
+  display: flex;
+  align-items: center;
+  z-index: 0;
+`;
+
+const textAreaStyle = css`
+  width: 100%;
+  min-height: ${spacing[6]}px;
+  resize: none;
+  border-radius: 4px;
+  padding-left: 12px;
+  font-size: 14px;
+  font-weight: normal;
+  font-family: ${fontFamilies.default};
+  z-index: 1;
```

Comment:
The border for the textArea doesn't currently match textInput. We should match the static and focus state border colors exactly.

Reply:
🤔 In light mode: 
interactionRing: uiColors.gray.light2,
interactionRingFocus: '#9dd0e7', (same across both components)

In dark mode: 
interactionRing: uiColors.gray.dark1,
interactionRingFocus: uiColors.blue.base,

Reply:
Not the interaction rings, the borders themselves.